libvcx/src/connection.rs

```diff
@@ -1374,7 +1129,7 @@ pub mod tests {
     #[test]
     #[cfg(feature = "general_test")]
     fn test_parse_acceptance_details() {
```

Comment:
This needs to be refactored to use V2 protocol, along with `parse_acceptance_details`.